PHP parse scripts/Modified_php_parse_scripts_by_Baertram/DumpVars/DumpVars_main.lua

```diff
@@ -1,97 +1,96 @@
-addonVars.addonVersion		   		= 0.01 -- Changing this will reset SavedVariables!
-addonVars.addonSavedVariablesName	= "DumpVars_Dump"
-addonVars.gAddonLoaded				= false
-
-local settings = {}
-settings.constants = {}
-settings.sounds = {}
-
-local function DumpVarsNow()
-	d("==================================")
-	d("[DumpVars - Dumping constants now]")
-    --Dump the constants values
-	local cnt = 0
-    if DumpVars and DumpVars.constantsToDump then
-        local const2Dump = DumpVars.constantsToDump
-        if const2Dump ~= nil then
-			settings.constants = {}
-	        for var2DumpStr, varDumped in pairs(const2Dump) do
-	            if var2DumpStr ~= nil then
-	                settings.constants[tostring(var2DumpStr)] = varDumped
-                    cnt = cnt + 1
-	            end
-	        end
-		    d(">dumped " .. tostring(cnt) .. " constants & their values!")
-        else
-		    d(">no constants wer dumped!")
-        end
-    end
-
-    --Dump the sounds afterwards too
-	cnt = 0
-	d("[DumpVars - Dumping SOUNDS now]")
-	if SOUNDS ~= nil then
-        settings.sounds = {}
-    	for soundName, soundFile in pairs(SOUNDS) do
-        	--settings.sounds[soundName] = soundFile
-        	settings.sounds[soundName] = ""
-            cnt = cnt + 1
-	    end
-        d(">dumped " .. tostring(cnt) .. " SOUNDS!")
-		d("DumpVars - ReloadUI will take place in 3 seconds")
-	    zo_callLater(function()
-            d("DumpVars - Reloading UI now!")
-        	ReloadUI()
-        end, 3000)
-    end
-end
-
-local function command_handler(arg)
-    arg = string.lower(arg)
-	if(arg == "") then
-    	DumpVarsNow()
-	end
-end
-
---Register the slash commands
-local function RegisterSlashCommands()
-    -- Register slash commands
-	SLASH_COMMANDS["/dumpvars"] = command_handler
-	SLASH_COMMANDS["/dv"] 		= command_handler
-end
-
-local function DumpVars_Loaded(eventCode, addOnName)
-	local defaults = {}
-
-	--Is this addon found?
-	if(addOnName ~= addonVars.gAddonName) then
-        return
-    end
-	--Unregister this event again so it isn't fired again after this addon has beend reckognized
-    EVENT_MANAGER:UnregisterForEvent(addonVars.gAddonName, EVENT_ADD_ON_LOADED)
-	addonVars.gAddonLoaded = false
-
-	--Register the slash commands
-	RegisterSlashCommands()
-
-    --Load the saved variables
-	settings = ZO_SavedVars:NewAccountWide(addonVars.addonSavedVariablesName, addonVars.addonVersion, "Settings", defaults)
-
-    d("DumpVars - Loaded")
-	addonVars.gAddonLoaded = true
-end
-
-local function DumpVars_Initialized()
-	EVENT_MANAGER:RegisterForEvent(addonVars.gAddonName, EVENT_ADD_ON_LOADED, DumpVars_Loaded)
-end
-
---------------------------------------------------------------------------------
---- Call the start function for this addon to register events etc.
---------------------------------------------------------------------------------
-DumpVars_Initialized()
+--DumpVars - Dumps variables from the game to the SavedVariables
```

Comment:
> Oh I see, you exchanged ReloadUI with Logout. Please revert that, a relaod uI is totally enough to write the data to SV file on disk!

Yes, but at least for me, /reloadui is a guaranteed server time out, the goal of which I'm 90% sure was just to make sure the SavedVariables updated, and the instructions said to follow it up with a logout anyways (which should also safely update the SavedVariables). /logout gives the user 10 seconds to cancel via pop-up if they don't want to.

Reply:
> Could you please check which file encoding and line breaks the original files use and keep it! It makes it really hard to see real differences if "all" is changed due to new line breaks type (windows vs. unix) used :-(

I had to disable `git config --global core.autocrlf` (i.e. `git config --global core.autocrlf false` which was automatically committing files with Unix line endings lol
You might want to set a .gitattributes file (https://docs.github.com/en/get-started/getting-started-with-git/configuring-git-to-handle-line-endings#per-repository-settings) with `text eol=crlf`?

Reply:
> > Oh I see, you exchanged ReloadUI with Logout. Please revert that, a relaod uI is totally enough to write the data to SV file on disk!
> 
> Yes, but at least for me, /reloadui is a guaranteed server time out, and the instructions said to follow it up with a logout anyways (which should also safely update the SavedVariables). /logout gives the user 10 seconds to cancel via pop-up if they don't want to, unlike /reloadui which takes a lot of time for me before logging me out anyways.

I never get any timeout on a reloadui? If you get that your SavedVAriables files in total of loaded addons are too big and you really should disable ALL other addons and only load vanilla ESO if you do a DumpVars either way!
I'll accept that change but will change it back to reloadui, it works 100% since years for me.
If you experience issues clean up your SV folder, alow file access in antivirus tools to your live/SavedVariables and /live folders and disable other addons before using DumpVars please.